klog.go

```diff
@@ -800,10 +800,19 @@ func kvListFormat(b *bytes.Buffer, keysAndValues ...interface{}) {
 			v = missingValue
 		}
 		b.WriteByte(' ')
-		if _, ok := v.(fmt.Stringer); ok {
+
+		var stringLike bool
```

Comment:
Can you avoid additional variable `stringLike`? `ObjectRef` should implement `fmt.Stringer`

Reply:
Type `string` seems no `String()` method

Reply:
Sure, I think code would be cleaner if we avoided additional variable. Maybe just put log call in switch

Reply:
I get this.